packages/controls/src/widget_image.ts

```diff
@@ -39,6 +39,19 @@ export class ImageView extends DOMWidgetView {
     this.update(); // Set defaults.
   }
 
+  generateMimeBundle() {
+    return {};
+  }
+
+  async generateMimeBundleExtra() {
+    const view: ImageView = await this.model.widget_manager.create_view<
```

Comment:
Now that it's in the view, you don't need to create another view anymore, and this could be the `generateMimeBundle` implementation instead of the extra's.

Reply:
Ah yes, good catch!